CompetitionTemplate/src/competition/subsystems/arm/ArmSubsystem.java

```diff
@@ -79,6 +82,10 @@ public boolean isArmAtMaximumHeight() {
         //return upperLimitSwitch.get();
     }
     
+    public boolean isArmAtMaxAngleHeight(){
```

Comment:
setArmMotorPower should use this boolean to determine if it should stop your motor - look at the if(enableSafeArmOperation.get())